src/table/styles.scss

```diff
@@ -152,6 +154,7 @@ filter search icon.
   margin-top: -15px;
   bottom: 0;
   width: 100%;
+  z-index: 15; // Higher than sticky columns
```

Comment:
There are a few z-indices, I saw `11`, `12`, `15`. Would it make sense to document this somewhere in the comments?

Reply:
Because it is on `position:fixed` it participates in page-global stacking context and should be documented here: https://cloudscape.design/get-started/dev-guides/z-index/

Also, all sticky elements have the index >800, should this one go into the same range?